_targets/meta/meta

```diff
@@ -0,0 +1,12 @@
+name|type|data|command|depend|seed|path|time|size|bytes|format|repository|iteration|parent|children|seconds|warnings|error
+clean_func|function|9f930f1c7268ccef|||||||||||||||
+diag_func|function|26ad1922e2f48204|||||||||||||||
```

Comment:
I think the step you may have missed that is causing this file to be committed is you did not add `_targets/*` to your top-level gitignore. Neither this file nor `_targets/.gitignore` should be comitted!